Formula/cifer.rb

```diff
@@ -16,6 +16,13 @@ class Cifer < Formula
     sha256 cellar: :any_skip_relocation, yosemite:      "bde7d97d9ef2a07c481ff8c5ec717fb2ec455fdef864db2a1a7b3056aa1934d2"
   end
 
+  on_linux do
+    depends_on "readline"
+
+    # Fix order of linker flags for GCC
+    patch :DATA
```

Comment:
Ideally this should be reported upstream.

Reply:
Not sure if the project is maintained. Last update was in 2009.

Code base is still only on Google Code, which was archived back in 2016.
I don't even know if there is a way to report issues anymore

Reply:
Let's merge this as-is then, as long as it builds. We will probably deprecate this formula one day if it get's too complicated to maintain.